pkgs/by-name/hc/hcp/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  gitUpdater,
+  hcp,
+}:
+
+buildGoModule rec {
+  pname = "hcp";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-YOOaQh1OsRn5EV9RmUdWWdHx5bMFC+a1qFzUGb6lpew=";
+  };
+
+  vendorHash = "sha256-/Nf180odZB5X3Fj4cfz0TdYEfGKtkkh4qI9eRfz+meQ=";
+
+  preCheck = ''
+    export HOME=$TMPDIR
+  '';
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
```

Comment:
I'm not sure, but I think the gitUpdater does *not* update the vendorHash.

Probably the nix-update-script could be used, I think it has a different parameter for version prefixes.